style.css

```diff
@@ -0,0 +1,575 @@
+/* Global Reset */
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Global Declarations */
+
+html {
+    font-family: sans-serif;
+}
+
+span {
+    color: #26a7de;
+}
+
+.active {
+    color: #26a7de;
+}
+
+/* Outer Container */
+
+main {
+    display: grid;
+    width: 100%;
+    height: 100%;
+    grid-template:
+    "nav  home  followers" 100vh /
+    minmax(320px, 380px)   2fr   minmax(320px, 380px)
+    ;
+    padding: 0 7rem;
+}
+
+/* Nav */
+nav {
+    display: grid;
+    grid-area: nav;
+    grid-template: 55px 2fr 1fr 100px / minmax(315px, 1fr);
+    padding: 1rem 0;
+}
+
+.nav-logo {
+    display: grid;
+    align-items: center;
+}
+
+/* Nav Index */
+
+.nav-index {
+    display: grid;
+    padding: 1rem 0;
+    min-height: 368px;
+}
+
+.nav-index-inner {
+    display: grid;
+    grid-template: 1fr / 32px 1fr;
+    align-items: center;
+}
+
+.nav-index-inner:hover {
+    transition: all ease-in 200ms;
+    color: #26a7de;
+    cursor: pointer;
+}
+
+.nav-index-inner > h2 {
+    padding-left: 0.4rem;
+}
+
+.nav-index-inner > i {
+    font-size: 1.4rem;
+}
+
+.nav-button > button{
+    color: #fff;
+    font-size: 1.2rem;
+    background-color: #26a7de;
+    border: #26a7de;
+    border-radius: 30px;
+    padding: 1rem;
+    width: 267.5px;
+}
+
+.nav-button > button:hover, .home-main-tweet-out-bottom-right > button:hover {
+    transition: all ease-in 100ms;
+    background-color: #26a7de;
+    cursor: pointer;
+    opacity: 0.8;
+}
+
+/* Nav Profile */
+
+.nav-profile {
+    display: grid;
+    grid-template: 1fr / 80px 1fr 80px;
+    padding: 0.7rem 0;
+    width: 320px;
+}
+
+.nav-profile-img, .follower-profile-img {
+    display: grid;
+    place-items: center;
+}
+
+.nav-profile-img-pic, .follower-profile-img-pic {
+    background-image: url(images/assets/pokeball.gif);
+    background-size: cover;
+    border: 1px solid #fff;
+    border-radius: 50%;
+    height: 55px;
+    width: 55px;
+    overflow: hidden;
+}
+
+.nav-profile-info, .follower-profile-info {
+    display: grid;
+    align-items: center;
+    padding: 0.6rem 0;
+}
+
+.nav-profile-info > p, .follower-profile-info > p {
+    font-weight: bold;
+}
+
+.nav-profile-info > span {
+    color: gray;
+}
+
+.nav-profile-more {
+    display: grid;
+    place-items: center;
+    padding-right: 0.75rem;
+}
+
+/* Home */
+.home {
+    display: grid;
+    grid-area: home;
+    grid-template: auto 1fr / 1fr;
+    padding-top: 1rem;
+    border: 1px solid #aab8c260;
+    min-width: 492px;
+}
+
+/* Tweet Bar */
+
+.home-main {
+    display: grid;
+    grid-template: 55px 140px / 1fr;
+    border-bottom: 14px solid #E1E8ED60; 
+}
```

Comment:
Not particularly, 1rem is refrencing the root font-size (which by default is 16px). I have used it for quite a while so that's why  I used it! (Might not be good practice idk)